### ISSUE TITLE
Support Selected Interface

### ISSUE THREAD

**Issue by py7hon** [vergoh/vnstat-docker#16]
Can you add the options **-i/-iface** for the selected interface ?

**Comment by vergoh** [comment]
Could you provide more details regarding (or an example of) the behaviour or use case you are looking for?

I'm not exactly sure yet what the missing feature is based on your request as `vnstat` in the container does expose `-i` / `--iface` to the command line.

**Comment by py7hon** [comment]
I mean like on `docker-compose.yml` having a Environment variables like `INTERFACE=eth0` 

```yml
version: "3.7"
services:

  vnstat:
    image: vergoh/vnstat:latest
    container_name: vnstat
    restart: unless-stopped
    network_mode: "host"
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /etc/timezone:/etc/timezone:ro
      - vnstatdb:/var/lib/vnstat
    environment:
      - HTTP_PORT=8685
      - INTERFACE=eth0 #like this
      - HTTP_BIND=*
      - HTTP_LOG=/dev/stdout
      - LARGE_FONTS=0
      - CACHE_TIME=1
      - RATE_UNIT=1
      - INTERFACE_ORDER=0
      - DARK_MODE=0
      - PAGE_REFRESH=0

volumes:
  vnstatdb:
  ```

**Comment by vergoh** [comment]
The use case you are having is that there are multiple interfaces and the automatic selection isn't picking up the interface you'd want to see by default?

**Comment by py7hon** [comment]
yeah i want select the interface but not all interface by default

**Comment by vergoh** [comment]
I could rather easily create and map the `INTERFACE` environment variable to configure `Interface` in the configuration file during container startup. However, if you have multiple interfaces then executing `vnstat` without any parameters would still result in the output containing all interfaces to be shown. That `Interface` configuration applies to only those outputs that have the ability of showing only one interface at time, such as `-5`, `-h`, `-d`, `-m` etc. Would this functionality be sufficient for you?

**Comment by vergoh** [comment]
I think I can make also the configurability of the default (no parameters) output happen with https://github.com/vergoh/vnstat/commit/79b5b9cfb162f0350dcd52035d12ec86319f9fa0 + also adding `QUERY_MODE` in addition to `INTERFACE`. That way, assuming `QUERY_MODE` is set to the new `4`, the summary output for interface defined with `INTERFACE` will get shown without any other interfaces getting listed even if the database would have more. I'm not sure if this is what you are after but this could still be useful for someone else too.

**Comment by vergoh** [comment]
The environment variables `INTERFACE` and `QUERY_MODE` are now available and map to `Interface` and `QueryMode` configuration settings. Note that `QueryMode` with value `4` is available starting from vnStat 2.12 so you'll have to use the `:dev` container image until version 2.12 has been released (assuming this it the feature you were looking for), after which the addition will also be available in `:latest` container image.
